src/main/kotlin/gg/skytils/skytilsmod/features/impl/spidersden/RainTimer.kt

```diff
@@ -28,18 +28,20 @@ import gg.skytils.skytilsmod.utils.graphics.colors.CommonColors
  * @link https://github.com/PikaFan123/rain-timer
  */
 object RainTimer {
-    var nextRain = 1596552707000
+    var nextRain = 1727548500000 // Unix-Time of a past rain event start in milliseconds
+    var eventCycleTime = 3600000 // Time between two rain event starts in milliseconds
```

Comment:
constants should be declared with `const val` for compiler optimizations

Reply:
Changes have been made to resolve change request